src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/TaxRate.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<resources xmlns="https://api-platform.com/schema/metadata"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata https://api-platform.com/schema/metadata/metadata-2.0.xsd"
+>
+    <resource class="%sylius.model.tax_rate.class%" shortName="TaxRate">
+        <attribute name="route_prefix">admin</attribute>
+
+        <attribute name="normalization_context">
+            <attribute name="groups">
+                <attribute>tax_rate:read</attribute>
+            </attribute>
+        </attribute>
+
+        <attribute name="denormalization_context">
+            <attribute name="groups">
+                <attribute>tax_rate:create</attribute>
+            </attribute>
+        </attribute>
```

Comment:
```suggestion
```

It should be added directly under the create endpoint. There is no point in extracting it if it should apply to only one route. Also, how will it behave on update action? Will we have both of the serialization groups or only one?

Reply:
You're right. I missed that during refactoring the previous PR. It'll be pushed soon.